prow/config/config_test.go

```diff
@@ -344,10 +344,11 @@ func TestGetGCSBrowserPrefix(t *testing.T) {
 func TestDecorationRawYaml(t *testing.T) {
```

Comment:
I think these test cases would make more sense in `TestValidConfigLoading`. That is also set up to test loading from multiple files which we need to test that strict validation works for job config.